mapper/tedge_mapper/src/sm_c8y_mapper/tests.rs

```diff
@@ -255,23 +183,27 @@ async fn mapper_publishes_software_update_status_and_software_list_onto_c8y_topi
 
     // Expect `116` and `503` messages with correct payload have been received on `c8y/s/us`, if no msg received for the timeout the test fails.
     let mut received_status_successful = false;
-    let mut received_software_list = false;
+
     for _ in 0..2 {
         match tokio::time::timeout(TEST_TIMEOUT_MS, subscriber.next()).await {
             Ok(Some(msg)) => {
                 dbg!(&msg.payload_str().unwrap());
                 match msg.payload_str().unwrap() {
-                    "503,c8y_SoftwareUpdate\n" => received_status_successful = true,
-                    "116,m,::apt,https://foobar.io/m.epl\n" => received_software_list = true,
```

Comment:
* The comment is now misleading. We expect just a `503`.
* The for loop is no more useful.